.buildkite/pipeline.yml

```diff
@@ -31,3 +31,8 @@ steps:
     commands:
       - "cd snapshotter"
       - "sudo make test EXTRAGOARGS='-v -count=1'"
+
+  - label: ':running_shirt_with_sash: runtime tests'
+    commands:
+      - "cd runtime"
```

Comment:
Why not to run all tests from root folder? So we don't need to update this if/when we add any other packages with tests (for example `agent` is going to have tests eventually)

Reply:
Summarizing our offline discussion: separate steps in Buildkite run in parallel, helping to increase utilization of our build fleet and decrease the test execution time.  Separate steps also gives granular status on the Buildkite status page, so it's easier to dive in and find where a test is failing.